Code/Tools/AssetProcessor/native/ui/ProductAssetTreeModel.cpp

```diff
@@ -205,7 +208,8 @@ namespace AssetProcessor
             {
                 if (!modelIsResetting)
                 {
-                    QModelIndex parentIndex = createIndex(parentItem->GetRow(), 0, parentItem);
+                    QModelIndex parentIndex = parentItem == m_root.get() ? QModelIndex() : createIndex(parentItem->GetRow(), 0, parentItem);
+                    Q_ASSERT(checkIndex(parentIndex));
```

Comment:
This would explain those Qt Asserts that occur in the AssetProcessor when running Debug